cmd/terramate-lsp/main.go

```diff
@@ -44,17 +60,24 @@ func main() {
 	// TODO(i4k): implement other modes.
 	if *modeFlag != "stdio" {
 		fmt.Println("terramate-lsp only supports stdio mode")
-		os.Exit(0)
+		os.Exit(1)
 	}
 
+	configureLogging(*logLevelFlag, *logFmtFlag, os.Stderr)
 	runServer(&readWriter{os.Stdin, os.Stdout})
 }
 
 func runServer(conn io.ReadWriteCloser) {
+	logger := log.With().
+		Str("action", "main.runServer()").
+		Logger()
+
+	logger.Trace().Msg("Creating context for UNIX signals.")
```

Comment:
```suggestion
	logger.Trace().Msg("Creating context for OS signals.")
```

Or something else since the signals are not UNIX specific (like os.Interrupt/Kill).